### ISSUE TITLE
Buyer/Seller

### ISSUE THREAD

**Issue by janbro** [lustephen/Project-Team-28#1]
Buyer and Seller class should implement off a base class so it can be stored inside Account under one variable type.  Additionally a few of their methods are the same.
eg. BuySell buyer = new Buyer(); BuySell seller = new Seller();

**Comment by PatrickBarber** [comment]
Yeah sorry I am doing a lot of changes this was just a layout. The base class will be the person class. Also I will be creating a 3rd class where I'll do the calculations instead of in the buyer seller class

**Comment by janbro** [comment]
Ok, no problem. I wasn't sure

**Comment by janbro** [comment]
I was just about to go through and try to make some open issues with what needs to be done

**Comment by janbro** [comment]
When you commit the changes for this can you include in your commit message "fixes #1", where #1 is the issue number. More info [here](https://help.github.com/articles/closing-issues-via-commit-messages/).

**Comment by PatrickBarber** [comment]
Made the buyer and seller inherit from the person classes and a few other changes. fixes#1

**Comment by janbro** [comment]
You're supposed to include "fixes #1" in the commit message, not the issue comment, eg.
```
> git add *
> git commit -m "Buyer/Seller changes. fixes #1"
> git push -u origin master
```
I'll go ahead and mark this issue closed
